includes/templates/class-listing-renew-package-page.php

```diff
@@ -19,6 +19,22 @@
  */
 class Listing_Renew_Package_Page extends Listing_Renew_Page {
 
+	/**
+	 * Class initializer.
+	 *
+	 * @param array $meta Class meta values.
+	 */
+	public static function init( $meta = [] ) {
+		$meta = hp\merge_arrays(
+			[
+				'label' => hivepress()->translator->get_string( 'listing_packages' ) . ' (' . esc_html__( 'Renew Listings', 'hivepress-paid-listings' ) . ')',
+			],
+			$meta
+		);
+
+		parent::init( $meta );
+	}
+
 	/**
 	 * Class constructor.
 	 *
```

Comment:
Please make sure that the listing packages block below respects the category template context. Add a new category-specific package, then create a custom template for Select Package page in HivePress/Templates, add the page title, listing packages, and change details link. Then try adding a new listing, the listing package on the Select Package step should appear for its specific category only.

Also, check if the listing packages block allows all the settings while creating a custom template, if it has the Category setting then it should be hidden (you can provide a list of allowed settings here in the block parameters), since this setting doesn't make sense - the block will always catch up the current listing category context.

Reply:
I've checked and everything is ok